src/scripts/exclusion_table_creation.py

```diff
@@ -276,23 +318,34 @@ def expand_intensional_exclusions(
     return df_results
 
 
-def run(
-    onto_path: str, exclusions_path: str, mirror_signature_path: str, component_signature_path: str,
-    config_path: str, outpath_txt: str, outpath_robot_template_tsv: str, save=CONFIG['save']
+def exclusion_table_creation(
+    onto_path: str, select_intensional_exclusions_path: str, mirror_signature_path: str, component_signature_path: str,
+    config_path: str, outpath_txt: str, outpath_robot_template_tsv: str, use_cache: bool = CONFIG['use_cache'],
+    save: bool = CONFIG['save']
 ) -> Union[Dict[str, pd.DataFrame], None]:
     """Run"""
     # Prefixes
     with open(config_path, 'r') as stream:
-        onto_config = yaml.safe_load(stream)
-    prefix_uri_map = onto_config['prefix_map']
+        onto_config: Dict = yaml.safe_load(stream)
+        prefix_uri_map: Dict = onto_config['prefix_map']
+    prefix_uri_map |= {'MONDO': 'http://purl.obolibrary.org/obo/MONDO_'}  # 'MONDO' for exclusion reasons
 
-    # Get excluded terms
+    # Get exclusions
+    # - Get intensional exclusions and expand them
     expanded_intensional_exclusions_df = expand_intensional_exclusions(
-        onto_path=onto_path, exclusions_path=exclusions_path, prefix_map=prefix_uri_map)
+        onto_path=onto_path, exclusions_path=select_intensional_exclusions_path, prefix_map=prefix_uri_map,
+        use_cache=use_cache)
+    # - Get non-inclusions
     non_inclusions_df = get_non_inclusions(
         mirror_signature_path=mirror_signature_path, component_signature_path=component_signature_path,
         prefix_map=prefix_uri_map)
-    df_results = pd.concat([expanded_intensional_exclusions_df, non_inclusions_df])
+    # - Get by triples with MONDO:exclusionReason predicate
```

Comment:
@nicolevasilevsky @matentzn @sabrinatoro Per exclude [non-diseases from OMIM slurp #110](https://github.com/monarch-initiative/mondo-ingest/issues/110), non-diseases are now excluded from OMIM.

#### `MONDO:excludeNonDisease` and `MONDO:excludeGene`
_However_, I just want you guys to know that there are effectively 0 new exclusions from _this_ change. This is because, as far as I can determine, the only terms I can be sure are non-diseases are genes. And, we're already excluding all of these with this single line in `config/omim_exclusions.tsv`:
```tsv
term_id	term_label	exclusion_reason	exclude_children
SO:0000704	gene	MONDO:excludeGene	TRUE
```

See this [related discussion in my OMIM PR](https://github.com/monarch-initiative/omim/pull/91#discussion_r1111318316) where I go over which symbols indicate non-disease. I'd like to confirm that I have this correct.

#### `MONDO:excludeTrait`
There _are_ new exclusions though from new `MONDO:excludeTrait` entries. This was the result of [OMIM terms without any symbols should be on the exclude list #169](https://github.com/monarch-initiative/mondo-ingest/issues/169).